lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -385,17 +374,17 @@ prefilterBlock b u0 = runState $ do
         , amount = Quantity amt
         }
     applyTx
-        :: (IsOurs s Address, IsOurs s ChimericAccount)
+        :: (IsOurs s Address)
         => ([(Tx, TxMeta)], UTxO)
         -> Tx
         -> State s ([(Tx, TxMeta)], UTxO)
     applyTx (!txs, !u) tx = do
         ourU <- state $ utxoOurs tx
         let ourIns = Set.fromList (inputs tx) `Set.intersection` dom (u <> ourU)
         let u' = (u <> ourU) `excluding` ourIns
-        ourWithdrawals <- mapMaybeM ourChimericAccount $ Map.toList $ withdrawals tx
+        let wdrls = fromIntegral . getCoin <$> Map.elems (withdrawals tx)
```

Comment:
could you explain this change?

Reply:
Yes. Since we can now have withdrawals in our wallet that aren't from our chimeric account (but will still contribute  positively to the input balance of a transaction), we need to account for pretty much all withdrawals in transactions (or otherwise, more complicated, "remember" all chimeric account we've made withdrawals from...). 

This will work fine until we introduce multisig (where some inputs will not be ours... right now, if one input is ours, then they all are so the problem is quite simplified). By then, we'll have to review this.